remove iup from stdlib in contrast to #15828 (#15830)
nim-lang/Nim: changelog.md

- Added `randState` template that exposes the default random number generator. Useful for library authors.

> - Removed deprecated `iup` module from stdlib, they have already moved to [nimble](https://github.com/nim-lang/iup).

Author: ``
```suggestion
- Removed deprecated `iup` module from stdlib, it has already moved to [nimble](https://github.com/nim-lang/iup).
```
